internal/hud/hud.go

```diff
@@ -161,19 +163,29 @@ func (h *Hud) handleScreenEvent(ctx context.Context, dispatch func(action store.
 				dispatch(view.AppendToTriggerQueueAction{
 					Name: selected.Name,
 				})
+			case r == '1':
```

Comment:
We forgot to update these keys to reflect the new order of tabs!

Reply:
updated!